### PR TITLE
Project description final

### PR DESCRIPTION
**Pull request by Luey-A**
Project Description component update after using i18next and fixing the css classes
- team work image
- project description header and paragraph

Screen Shot 2020-11-17 at 23 35 14

### CODE REVIEW

File: src/components/ProjectDescription/index.jsx [hunk 1]
```diff
@@ -1,28 +1,25 @@
 import React from "react";
 import "./index.css";
-import gift from "./gift.jpg";
 import { Container, Row, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import gift from "./gift.jpg";
```
Comment (Ammar-64) [inline src/components/ProjectDescription/index.jsx]: Indicate that this an image, by adding img to the end so the reader would understand it

**Review by Ammar-64 — APPROVED** [review]
LGTM, there’s a conflict please resolve it with dev, and you’re good to go

**Review by betulerkoc — APPROVED** [review]

File: src/components/ProjectDescription/index.jsx (hunk 1 above)
Comment (Luey-A) [reply src/components/ProjectDescription/index.jsx]: PTAL

**Comment by Luey-A** [comment]
> LGTM, there’s a conflict please resolve it with dev, and you’re good to go

PTAL

**Pull request merged**